models/sipnet/R/read_restart.SIPNET.R

```diff
@@ -88,6 +88,11 @@ read_restart.SIPNET <- function(outdir, runid, stop.time, settings, var.names, p
     names(forecast[[length(forecast)]]) <- c("TotLivBiom")
   }
   
+  if ("LAI" %in% var.names) {
+    forecast[[length(forecast) + 1]] <- ens$LAI[last]  ## m2/m2 
+    names(forecast[[length(forecast)]]) <- c("LAI")
```

Comment:
No need for `c` here -- just `"LAI"` does exactly the same thing.
```suggestion
    names(forecast[[length(forecast)]]) <- "LAI"
```

Also, super minor thing, but FWIW, where you have `ens$LAI[last]` -- if `last` is always supposed to be a scalar, I would use double brackets: `ens$LAI[[last]]`. `[[` subsetting always returns a single object, and throws a nice loud error if you give a non-scalar input. This makes it easier to catch bugs, and makes the intent of the code clearer, at least IMHO.